packages/styles/src/card.scss

```diff
@@ -388,6 +390,10 @@ $fd-card-header-outline-offset: 0.0625rem !default;
     @include fd-reset();
 
     padding-bottom: 0.5rem;
+
+    h3 {
+      font-size: var(--sapFontSize);
+    }
```

Comment:
I don't think this is correct. Using `h3` brings additional margins top and bottom to the element, which completely deviates from the visual design:
<img width="367" alt="Screenshot 2023-11-07 at 12 11 37 PM" src="https://github.com/SAP/fundamental-styles/assets/39598672/f7fe92d2-4908-49e0-870c-cd01547f86d3">

Reply:
The size of the title is something the user should specify. What comes from us is only the spacing below the container. In my opinion, the comment in the figma should be ignored.

<img width="513" alt="Screenshot 2023-11-07 at 12 15 02 PM" src="https://github.com/SAP/fundamental-styles/assets/39598672/2d1fd1d8-59f4-415c-b1f8-24b67d1ef74c">